docs/rulesets.md

```diff
@@ -1,41 +1,128 @@
+
+While running it with this object will succeed:
+
+```json
+{
+  "tags": [{
+    "name": "animals",
+    "description": "come in all shapes and sizes"
+  }]
+}
+```
+
+### Adding to the recommended OpenAPI rules
+
+Spectral comes with two built in rulesets - one for OpenAPI v2 (`spectral:oas2`), and one for OpenAPI v3 (`spectral:oas3`). Use the `extends` property in your ruleset file to build upon or customize other rulesets.
+
+**spectral.yml**
+
+```yaml
+extends: spectral:oas2
+rules:
+  my-rule-name:
+    description: Tags must have a description.
+    given: $.tags[*]
+    then:
+      field: description
+      function: truthy
+```
+
+The example above will apply the recommended rules from the built in OpenAPI v2 ruleset AND apply the custom `my-rule-name` rule.
+
+### Enabling specific OpenAPI rules
+
+Sometimes you might want to apply specific rules from another ruleset. Use the `extends` property, and pass `off` as the second argument in order to load add the rules from another ruleset, but disable them all by default. This allows you to pick and choose which rules you would like to enable.
+
+**spectral.yml**
+
+```yaml
+extends: [[spectral:oas2, off]]
+rules:
+  # This rule is defined in the spectral:oas2 ruleset. We're passing `true` to turn it on and inherit the severity defined in the spectral:oas2 ruleset.
+  operation-operationId-unique: true
+```
+
+The example above will run the two rules we enabled, since we pass `off` to disable all rules by default when extending the `spectral:oas2` ruleset.
+
+### Disabling specific OpenAPI rules
+
+This example shows the opposite of the `Enabling specific OpenAPI rules` example. Sometimes you might want to enable all rules by default, and disable a few.
+
+**spectral.yml**
+
+```yaml
+extends: [[spectral:oas2, all]]
+
+# we could also just extend spectral:oas2 like this, which would only run the recommended rules (rather than every single rule)
+# extends: spectral:oas2
+
+rules:
+  operation-operationId-unique: false
 ```
 
-## Example Ruleset
+The example above will run all of the rules defined in the `spectral:oas2` ruleset (rather than the default behavior that runs just the recommended ones), with two exceptions - we turned `operation-operationId-unique` and `operation-2xx-response` off.
+
+### Changing the severity of a rule
 
-We currently support ruleset files in both `yaml` and `json` formats.
+**spectral.yml**
 
 ```yaml
-extends:
-  - https://acme.com/ruleset.json
-  - /path/to/ruleset.yaml
+extends: spectral:oas2
 rules:
-    rule-name:
-        given: $..parameters[*]
-        then:
-            field: description
-            function: truthy
+  operation-2xx-response: warn
 ```
 
+The example above will run the recommended rules from the `spectral:oas2` ruleset, but report `operation-2xx-response` as a warning rather than as an error (as is the default behavior in the `spectral:oas2` ruleset).
+
+Available severity levels are `error`, `warn`, `info`, and `off`.
+
 ## Rules
```

Comment:
https://github.com/stoplightio/spectral/blob/77cb641d7deea9c24dc672019efdd635f7ad3c88/docs/rulesets.md#rules it's updated, we should include human-readable severity levels (off, warn, error etc)